src/Contrib/Jaeger/SpanConverter.php

```diff
@@ -109,42 +110,50 @@ private function buildTags($tagPairs)
         return $tags;
     }
 
-    private function buildTag($key, $value)
+    private function buildTag(string $key, string $value)
     {
-        if (is_bool($value)) {
-            return new Tag([
-                'key' => $key,
-                'vType' => TagType::BOOL,
-                'vBool' => $value,
-            ]);
-        } elseif (is_string($value)) {
-            return new Tag([
-                'key' => $key,
-                'vType' => TagType::STRING,
-                'vStr' => $value,
-            ]);
-        } elseif (null === $value) {
-            return new Tag([
-                'key' => $key,
-                'vType' => TagType::STRING,
-                'vStr' => '',
-            ]);
-        } elseif (is_integer($value)) {
-            return new Tag([
-                'key' => $key,
-                'vType' => TagType::LONG,
-                'vLong' => $value,
-            ]);
-        } elseif (is_numeric($value)) {
-            return new Tag([
-                'key' => $key,
-                'vType' => TagType::DOUBLE,
-                'vDouble' => $value,
-            ]);
-        }
+        return new Tag([
+            'key' => $key,
+            'vType' => TagType::STRING,
+            'vStr' => $value,
+        ]);
+
+        //TODO - determine if any of this additional handling is still needed, or if it's all dead-code (i.e. $value must be a string at this point always)
+
+        // if (is_bool($value)) {
```

Comment:
This is the new commented out code. It seemed like in the current state $value is always a string, so I couldn't figure out how the rest of this code could be hit